crates/pathfinder/src/state/sync.rs

```diff
@@ -822,7 +826,15 @@ async fn l2_update(
             }
         }
 
-        transaction.commit().context("Commit database transaction")
+        transaction
+            .commit()
+            .context("Commit database transaction")?;
+
+        if let Some(sender) = websocket_txs {
+            _ = sender.new_head.send_if_receiving(header.into());
```

Comment:
We'll probably want to log if this fails

Reply:
I've added this, but also disable the sender afterwards to reduce spamming error logs. Let me know what you think of this.